### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by hobinjk-ptc**

### CODE REVIEW

File: src/device/onLoad.js [hunk 1]
```diff
@@ -188,43 +188,52 @@ realityEditor.device.onload = function () {
...
-                    } else {
-                        recordButton.classList.add('pointCloudButtonActive');
-                        recordButton.src = '../../../svg/recordButton3D-stop.svg';
-                        realityEditor.device.videoRecording.start3DVideoRecording();
-                    }
-                });
+            function enablePoseTracking() {
+                let bestWorldObject = realityEditor.worldObjects.getBestWorldObject();
+                if (!bestWorldObject || bestWorldObject.objectId === realityEditor.worldObjects.getLocalWorldId()) {
+                    enablePoseTrackingTimeout = setTimeout(enablePoseTracking, 100);
```
Comment (benptc) [inline src/device/onLoad.js:198]: Nice to have this

File: src/gui/threejsScene.js [hunk 2]
```diff
@@ -78,6 +79,12 @@ import { InfiniteGridHelper } from '../../thirdPartyCode/THREE.InfiniteGridHelpe
 
         addGroundPlaneCollider(); // invisible object for raycasting intersections with ground plane
 
+        let pmremGenerator = new THREE.PMREMGenerator(renderer);
+        pmremGenerator.compileEquirectangularShader();
+
+        let neutralEnvironment = pmremGenerator.fromScene(new RoomEnvironment()).texture;
+        scene.environment = neutralEnvironment;
```
Comment (benptc) [inline src/gui/threejsScene.js:86]: Is this analogous to adding an HDRI to the scene for lighting?

**Review by benptc — APPROVED** [review]
Nice updates!

File: src/gui/threejsScene.js (hunk 2 above)
Comment (hobinjk-ptc) [reply src/gui/threejsScene.js:86]: Yep! (had to look up what an HDRI is 🤣 )

**Pull request merged**